packages/js/customer-effort-score/src/components/customer-effort-score-tracks/index.js

```diff
@@ -173,49 +180,9 @@ function _CustomerEffortScoreTracks( {
 	);
 }
 
-_CustomerEffortScoreTracks.propTypes = {
-	/**
-	 * The action name sent to Tracks.
-	 */
-	action: PropTypes.string.isRequired,
-	/**
-	 * Additional props sent to Tracks.
-	 */
-	trackProps: PropTypes.object,
-	/**
-	 * The label displayed in the modal.
-	 */
-	title: PropTypes.string.isRequired,
-	/**
-	 * The label for the snackbar that appears upon survey submission.
-	 */
-	onSubmitLabel: PropTypes.string,
-	/**
-	 * The array of actions that the CES modal has been shown for.
-	 */
-	cesShownForActions: PropTypes.arrayOf( PropTypes.string ),
-	/**
-	 * Whether tracking is allowed or not.
-	 */
-	allowTracking: PropTypes.bool,
-	/**
-	 * Whether props are still being resolved.
-	 */
-	resolving: PropTypes.bool.isRequired,
-	/**
-	 * The age of the store in weeks.
-	 */
-	storeAgeInWeeks: PropTypes.number,
-	/**
-	 * Function to update options.
-	 */
-	updateOptions: PropTypes.func,
-	/**
-	 * Function to create a snackbar
-	 */
-	createNotice: PropTypes.func,
-};
-
+/**
```

Comment:
typescript complained about the HOC not having a typedef and this is a .js file so I converted proptypes to ts inline comment typedefs